lms/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "lms-signature"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+name = "lms"
+path = "lms/lib.rs"
```

Comment:
Hmm, this is a lot of directories named `lms`, i.e. `lms/lms/lms`. Perhaps use a standard `src` directory instead?